ui/admin/app/routes/scopes/scope/auth-methods.js

```diff
@@ -193,14 +193,15 @@ export default class ScopesScopeAuthMethodsRoute extends Route {
   /**
    * Adds an account claim map fragment to the passed OIDC `authMethod`.
```

Comment:
Can this comment be updated to reflect the changes?

Reply:
good catch, updated! thank you!